### DISCUSSION TITLE
Welcome to ZeroTier-GUI-Arch Discussions!

### DISCUSSION THREAD

**Discussion by aaron777collins** [aaron777collins/ZeroTier-GUI-Arch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wpetrowski** [comment]
Hey, just wanted to drop a quick note of appreciation. I'd been using zerotier on the deck for a little while, but found it flaky and hard to manage.  This makes it so much easier.  Thanks for making this!

**Reply by aaron777collins** [reply]
Wow thanks for appreciating the app. I'm glad you're loving it! Please let me know if you feel that it's missing anything and I'll try to add it. Thanks again for the support :)
